### PR TITLE
Link to individual comments from the audit log

### PR DESCRIPTION
**Pull request by kraih**
Alternative approach for #4652. Discussions on Slack about another comment related bug showed that it can be quite useful to have `job_id`/`group_id`/`parent_group_id` in the audit log.

Progress: https://progress.opensuse.org/issues/111060

### CODE REVIEW

**Comment by kraih** [comment]
Tests will be added too before i merge this PR.

**Review by perlpunk — COMMENTED** [review]
lgtm

**Review by okurz — APPROVED** [review]

**Review by Martchus — APPROVED** [review]

**Comment by kraih** [comment]
Now with tests.

File: t/ui/23-audit-log.t
Comment (okurz) [inline t/ui/23-audit-log.t:18]: Now I realized that you added this which we should avoid to save runtime and reduce inter-dependencies of tests. Well, you were right, we should have discarded review approvals after changes. At least here it would have prevented that ;) I will try to fix that.

**Pull request merged**
